### ISSUE TITLE
Auto-increase transaction timeout when debugger is attached

### ISSUE THREAD

**Issue by famod** [quarkusio/quarkus#19351]
### Description

After seeing #18668 I had this idea to also auto-increase transaction timeouts when debugger is attached.

This would be very useful primarily in dev and test mode.

### Implementation ideas

This should probably be an opt in in the form of a config property that defines the timeout value to increase to. If it's not set, no auto-increase takes place.

**Comment by mmusgrov** [comment]
@famod I don't think this issue requires input from team responsible for the JTA extension. Would you be able to remove the `area/narayana` label so that is no longer appears in our tracking tool. But if you do require input from narayana then do tag us.
